src/lib/os.pl

```diff
@@ -14,7 +14,8 @@
 
 :- module(os, [getenv/2,
                setenv/2,
-               unsetenv/1]).
+               unsetenv/1,
+			   pid/1]).
```

Comment:
Please consider using the whitespace conventions of the source files to make changes easier to review.

Reply:
That's always my intention when contributing to third-party projects. But I forgot to change my indentation preferences in this case. Thanks for noticing.

P.S. These issues can be avoided by adding a `.editorconfig` file to the root of this repo with the project preferences for each kind of file. Then any contributor using a text editor or IDE that supports [EditorConfig](https://editorconfig.org) will automatically use the project preferences.